app/src/main/java/com/example/randomusergenerator/view/components/SearchComponents.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.randomusergenerator.view.components
+
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import com.example.randomusergenerator.R
+
+@Composable
+fun ExpandableSearchIcon(onClick: () -> Unit) {
+    IconButton(onClick = onClick) {
+        Icon(Icons.Default.Search, contentDescription = stringResource(R.string.search))
+    }
+}
+
+@Composable
+fun SearchCloseButton(onClick: () -> Unit) {
```

Comment:
This could be in its own file.

Reply:
I wanted all components for the search to be in this class

Reply:
You can make them in separate files.

Reply:
okay